care/facility/api/serializers/shifting.py

```diff
@@ -323,8 +323,15 @@ def update(self, instance, validated_data):
         if (
             "status" in validated_data
             and validated_data["status"] == REVERSE_SHIFTING_STATUS_CHOICES["COMPLETED"]
+            and not has_facility_permission(user, instance.origin_facility)
         ):
-            discharge_patient(instance.patient)
```

Comment:
Why are we deleting this statement?

Reply:
Previously, We automatically discharged the patient when the shifting reason was completed but now once the shifting is done (this API), we show the discharge form in the frontend so that user can enter details like discharge reason, etc before discharging a patient.